src/main/java/com/fasterxml/jackson/datatype/jsr310/ser/InstantSerializerBase.java

```diff
@@ -37,13 +38,23 @@
 
     private final ToIntFunction<T> getNanoseconds;
 
+    private final Function<T, String> toIsoString;
+
+    protected InstantSerializerBase(Class<T> supportedType, ToLongFunction<T> getEpochMillis,
+                                    ToLongFunction<T> getEpochSeconds, ToIntFunction<T> getNanoseconds)
+    {
+        this(supportedType, getEpochMillis, getEpochSeconds, getNanoseconds, t -> t.toString());
```

Comment:
Same as a comment below: s/`t -> t.toString()`/`t::toString`/, compiler to confirm if syntax is correct.